web3.js/src/util/chunks.ts

```diff
@@ -0,0 +1,8 @@
+export function sliceIntoChunks(arr: any, chunkSize: number) {
+  const res = [];
+  for (let i = 0; i < arr.length; i += chunkSize) {
+    const chunk = arr.slice(i, i + chunkSize);
+    res.push(chunk);
+  }
+  return res;
+}
```

Comment:
Try turning trailing newlines on in your editor; the GitHub diff is trying to tell you that you're missing one by displaying this symbol.

<img width="147" alt="image" src="https://user-images.githubusercontent.com/13243/160073925-ff76c7fc-ac43-499c-ade1-273753669014.png">